meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -21,7 +22,7 @@ mkdir -p /data_disk/docker
 mkdir -p /data_disk/resin-data
 
 # Start docker with the created image.
-docker daemon -g /data_disk/docker -s btrfs &
+docker daemon -g /data_disk/docker &
```

Comment:
Yes, it has to be `aufs`. The underlying layer storage changes depending on this setting. The build server will have to support aufs unless we find another way to create this partitions